src/DGmethods/DGmodel_kernels.jl

```diff
@@ -439,7 +445,7 @@ function volumeviscterms!(bl::BalanceLaw, ::Val{dim}, ::Val{polyorder},
           ξ2x1, ξ2x2, ξ2x3 = vgeo[ijk, _ξ2x1, e], vgeo[ijk, _ξ2x2, e], vgeo[ijk, _ξ2x3, e]
           ξ3x1, ξ3x2, ξ3x3 = vgeo[ijk, _ξ3x1, e], vgeo[ijk, _ξ3x2, e], vgeo[ijk, _ξ3x3, e]
 
-          @unroll for s = 1:ngradstate
+          @unroll for s = 1:nG
```

Comment:
More verbose here as well. `ngradstate` was better than `nG`. Something that makes the function of the variable clearer.